syntax="proto3";

package igclient;

service IGService {
  rpc CreateIGTVVideo(CreateIGTVVideoRequest) returns (CreateIGTVVideoResponse);
}

message CreateIGTVVideoRequest {
  string igUsername = 1;
  string igPassword = 2;
  string title = 3;
  string caption = 4;
  string videoURL = 5;
  string thumbnailURL = 6;
}

message CreateIGTVVideoResponse {
  string id = 1;
}